awswrangler/athena/_read.py

```diff
@@ -139,9 +137,9 @@ def _fetch_parquet_result(
     else:
         ret = _add_query_metadata_generator(dfs=ret, query_metadata=query_metadata)
     paths_delete: List[str] = paths + [manifest_path, metadata_path]
-    _logger.debug("type(ret): %s", type(ret))
     if chunked is False:
         if keep_files is False:
+            _logger.debug("Deleting objects in S3 path: %s", paths_delete)
```

Comment:
I have already added a similar log call inside the `delete_objects` method in my own S3 logging PR so that others don't have to add calls like these

Reply:
Ok great, thanks, removed